### PR TITLE
Fix migrations and add test data

### PR DESCRIPTION
**Pull request by llmlks**
Even after this issue passes code review, do not merge to master. Need to first manually undo migrations on staging server, as the migration files have been changed

### CODE REVIEW

File: scripts/deploy-k8s.sh [hunk 1]
```diff
@@ -25,4 +25,6 @@ gcloud config set project emblica-212815
 gcloud config set compute/region europe-north1
 gcloud container clusters get-credentials emblica-cluster-1 --region europe-north1
 
-kubectl set image deployment/${K8S_DEPLOYMENT_NAME} ${K8S_DEPLOYMENT_NAME}=${DOCKER_IMAGE}:${TRAVIS_BUILD_NUMBER} -n rekrysofta
\ No newline at end of file
+kubectl set image deployment/${K8S_DEPLOYMENT_NAME} ${K8S_DEPLOYMENT_NAME}=${DOCKER_IMAGE}:${TRAVIS_BUILD_NUMBER} -n rekrysofta
+
+kubectl exec -it -n rekrysofta $(kubectl get pods -n rekrysofta | grep server | awk '{ print $1 }') -- bash -c 'npm run migrate:production'
```
Comment (josalmi) [inline scripts/deploy-k8s.sh]: Not the best idea as set image command starts killing the container and recreating a new one with new image. You could end up in situation where migrations are run in the old container using old migrations. If migrations don't have any effect if run multiple times the best way for now might be to run migrations on startup as part of npm start or something

File: server/db/migrations/20181124082120-add-recruiter-username-to-comments.js [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
 
 module.exports = {
-  up: (queryInterface, Sequelize) => {
+  up: async (queryInterface, Sequelize) => {
+    const applicationsComments = 'ApplicationComments'
+    const recruiterUsername = 'recruiterUsername'
+    const tableDefinition
+      = await queryInterface.describeTable(applicationsComments)
+
+    if (tableDefinition[recruiterUsername])
```
Comment (josalmi) [inline server/db/migrations/20181124082120-add-recruiter-username-to-comments.js]: This seems a bit hacky. Is the database modified somewhere else than migrations? You should know the state of database if migrations are run in order.

Comment (llmlks) [reply server/db/migrations/20181124082120-add-recruiter-username-to-comments.js]: I don't quite understand why, but every time the migrations are run, an error is produced for the migrations altering existing tables. This happens also when the applications is run for the first time, i.e. the database is created from scratch. It seems that each table is for some reason created with all columns during the first migration, where it is created, even though some of the columns should be introduced in later migrations. Maybe the model file is somehow used for determining the schema, although it should be done based on the migration files alone.

File: scripts/deploy-k8s.sh (hunk 1 above)
Comment (llmlks) [reply scripts/deploy-k8s.sh]: Ah, yes, I didn't think of that. Thanks for the suggestion!

File: server/db/migrations/20181124082120-add-recruiter-username-to-comments.js (hunk 2 above)
Comment (josalmi) [reply server/db/migrations/20181124082120-add-recruiter-username-to-comments.js]: I think that is exactly what is happening and is documented in sequelize documentation

Comment (llmlks) [reply server/db/migrations/20181124082120-add-recruiter-username-to-comments.js]: Finally understood the problem, thanks!

**Pull request merged**